qualang_tools/plot/Fitting/fitting.py

```diff
@@ -0,0 +1,134 @@
+from fitting_functions import *
+import itertools
+import json
+import matplotlib.pyplot as plt
+
+
+class Fit:
+    """
+    This class takes care of the fitting to the measured data.
+    All the fitting functions are taken from the `Fitting_functions` file.
```

Comment:
In python packages, from the user perspective, there are no "files".
If we want user to have access to the functions directly, then we can just rename it to "module". Otherwise it should be removed.